src/align.py

```diff
@@ -16,7 +20,30 @@ def align(x: str, y: str, edits: str) -> tuple[str, str]:
     ('ACCACAGT-CATA', 'A-CAGAGTACAAA')
 
     """
-    return "", ""
+    x_align = ''
+    y_align = ''
+    i = 0
+    j = 0
```

Comment:
For each edit, you will add a symbol to both `x_align` and `y_align`, so you don't need separate indices here. If the two strings are not the same length, then they are not aligned (and that goes for any prefix of `edits` as well).